src/server/api/routes/channels-router.js

```diff
@@ -0,0 +1,213 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const channelsController = require('../controllers/channels-controller');
+
+/**
+ * @swagger
+ * /modules:
+ *  post:
+ *    summary: Create a channel
+ *    description:
+ *      Will create a channel.
+ *    produces: application/json
+ *    parameters:
+ *      - in: body
+ *        name: channel
+ *        description: The channel to create.
+ *        schema:
+ *          type: object
+ *          required:
+ *              - created_at
+ *              - updated_at
+ *          properties:
+ *            title:
+ *              type: string
+ *            created_at:
+ *              type: string
+ *              format: date-time
+ *
+ *            updated_at:
+ *              type: string
+ *              format: date-time
+ *
+ *            deleted_at:
+ *              type: string
+ *              format: date-time
+ *
+ *    responses:
+ *      201:
+ *        description: Channel created
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.post('/', (req, res) => {
```

Comment:
You were not supposed to create a POST endpoint, just GET one.

Reply:
I didn't create, I copied from Gebremedhin's PR. I only add id, because I have error in swagger in get by id.